index.html

```diff
@@ -17,122 +17,150 @@
+        <div class="characteristics__picture characteristics__picture--left">
+          <img src="./assets/img/characteristics-2.png" alt="design" />
+        </div>
         <div class="characteristics__text">
-          <p> Easier decision making for</p>
-          <h2>Product Managers</h2>
-          <p>The Myspace page defines the individual, his or her characteristics, traits, personal choices and the
-            overall
-            personality of the person. </p>
-          <ul>
-            <li>Never worry about overpaying for your energy again. </li>
-            <li>We will only switch you to energy companies that we trust and will treat you right</li>
-            <li>We track the markets daily and know where the savings are.</li>
+          <p class="characteristics__p"> Easier decision making for</p>
+          <h2 class="characteristics__h2">Product Managers</h2>
+          <p class="characteristics__p">The Myspace page defines the individual, his or her characteristics, traits,
+            personal choices and the overall personality of the person. </p>
+          <ul class="characteristics__ul">
+            <li class="characteristics__li">Never worry about overpaying for your energy again. </li>
+            <li class="characteristics__li">We will only switch you to energy companies that we trust and will treat you
+              right</li>
+            <li class="characteristics__li">We track the markets daily and know where the savings are.</li>
           </ul>
         </div>
       </div>
 
       <div class="characteristics-row third">
         <div class="characteristics__text">
-          <p> Optimisation for</p>
-          <h2>Marketers</h2>
-          <p>Few would argue that, despite the advancements of feminism over the past three decades, women still face a
+          <p class="characteristics__p"> Optimisation for</p>
+          <h2 class="characteristics__h2">Marketers</h2>
+          <p class="characteristics__p">Few would argue that, despite the advancements of feminism over the past three
+            decades, women
+            still face a
             double standard when it comes to their behavior. </p>
-          <h4>Accessory makers </h4>
-          <p> While most people enjoy casino gambling, sports betting, lottery and bingo playing for the fun</p>
-          <h4 class="title--up">Alterationists</h4>
-          <p>If you are looking for a new way to promote your business that won’t cost you more money, </p>
-          <h4 class="title--last">Custom Design designers</h4>
-          <p>If you are looking for a new way to promote your business that won’t cost you more money, </p>
+          <h4 class="characteristics__h4">Accessory makers </h4>
+          <p class="characteristics__p"> While most people enjoy casino gambling, sports betting, lottery and bingo
+            playing for the fun
+          </p>
+          <h4 class="characteristics__h4 title--up">Alterationists</h4>
```

Comment:
title--up is quite confusing name
it doesn't give and idea what it's doing with title

Reply:
completed